apps/wallet/src/ui/app/pages/home/transfer-coin/TransferCoinForm/StepTwo.tsx

```diff
@@ -88,31 +88,48 @@ function StepTwo({
                     </div>
 
                     <div className={st.details}>
-                        <div className={st.txFees}>
-                            <div className={st.txInfoLabel}>Gas Fee</div>
-                            <div className={st.walletInfoValue}>
-                                {formattedGas} {GAS_SYMBOL}
-                            </div>
-                        </div>
-
-                        <div className={st.txFees}>
-                            <div className={st.txInfoLabel}>Total Amount</div>
-                            <div className={st.walletInfoValue}>
-                                {formattedTotal} {GAS_SYMBOL}
+                        {[
```

Comment:
Fine for now but every time I see patterns like this I think we should start breaking out components.